getting-set-up.qmd

```diff
@@ -23,7 +23,7 @@ The course relies on running stan through the `cmdstanr` **R** package, which it
 This requires a separate installation step:
 
 ```{r cmdstan_install, eval = FALSE}
-cmdstanr::install_cmdstan()
+cmdstanr::install_cmdstan(cores = 2)
```

Comment:
why 2?

Reply:
so that is compiles faster and under the assumption they will have at least two we can use?

Reply:
minor point of course but I'm not sure why we'd overrule the `cmdstanr` default here. Also, every extra argument adds complexity so I think we should only use what's definitely necessary (i.e. not this).

Reply:
the stan default is 1 core and so slow is the reason. People new to cmdstan won't know it exists. I don't really mind either way